.github/workflows/push-training-ecs.yml

```diff
@@ -32,14 +32,14 @@ on:
 
 env:
   AWS_REGION: "us-east-1" # set this to your preferred AWS region, e.g. us-west-1
-  ECR_REPOSITORY: "dlp-backend-image" # set this to your Amazon ECR repository name
-  ECS_SERVICE: "BackendService" # set this to your Amazon ECS service name
-  ECS_CLUSTER: "BackendCluster" # set this to your Amazon ECS cluster name
-  ECS_TASK_DEFINITION:
-    ".aws/training-task-definition.json" # set this to the path to your Amazon ECS task definition
```

Comment:
same idea here?

Reply:
same as above, just renamed `ECS_SERVICE` and `ECS_CLUSTER` (but deleted `ECS_TASK_DEFINITION`)

Reply:
same followup above

Reply:
<img width="1136" alt="image" src="https://github.com/DSGT-DLP/Deep-Learning-Playground/assets/47485510/44ce52e4-3c0f-4186-9126-a35368b22018">